src/components/WorkOrder/Appointment/PropertyDetails.js

```diff
@@ -10,12 +15,56 @@ const PropertyDetails = ({
   propertyReference,
   boilerHouseId,
 }) => {
+  const [alerts, setAlerts] = useState([])
+  const [alertsLoading, setAlertsLoading] = useState(false)
+  const [alertsError, setAlertsError] = useState()
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  const fetchAlerts = async () => {
+    const alertsResponse = await getAlerts(propertyReference)
```

Comment:
Given youre making major changes, could you take the opportinuty to convert it to typescript?